magenta/pipelines/statistics.py

```diff
@@ -129,7 +129,7 @@ def merge_statistics(stats_list):
   return list(name_map.values())
 
 
-def log_statistics_list(stats_list, logger_fn=tf.logging.info):
+def log_statistics_list(stats_list, logger_fn=tf.compat.v1.logging.info):
```

Comment:
Could you just change the import at the top of this file to: `import tensorflow.compat.v1 as tf`